runtime/src/main/java/com/flipkart/flux/metrics/impl/MetricsClientImpl.java

```diff
@@ -0,0 +1,35 @@
+package com.flipkart.flux.metrics.impl;
```

Comment:
Add Copyright, refer to any file for the copyright text. You can configure Intellij to add copyright on creation.
https://www.jetbrains.com/help/idea/2016.3/generating-and-updating-copyright-notice.html